docs/accessing_data.md

```diff
@@ -119,13 +120,57 @@ See the [getting started with ActiveData] guide for more details.
 
 ## Direct database access
 
-If there are any use-cases that are not possible via one of the above, we can provide read-only
+If the use-cases above aren't sufficient, we can provide read-only
```

Comment:
Could you add a note here that you cannot login to the application when using the replica? :)